reg/drone/physics/electricity/Source.0.1.uavcan

```diff
@@ -0,0 +1,27 @@
+# A generic source or sink of electric power (battery, turbogenerator, braking resistor, etc.).
+# Low-pass filtering should be applied to avoid aliasing effects (as is the case everywhere else).
+
+Power.0.1 load_power
+# Total instant power supplied to the load.
+# Positive current flows from the source to the load (discharge/production).
+# Negative current flows into the source (regeneration/consumption).
+
+uavcan.si.unit.energy.Scalar.1.0 available_energy
+# A pessimistic estimate of the amount of energy that can be reclaimed from the source in its current state.
+# This may be dependent on the state of charge/health (for batteries), temperature, load profile, humidity, etc.
```

Comment:
If we are making reference to conditions, we should have a service call to get this same information (just a note for the service namespace).

Reply:
This is going to be addressed by the BMS-specific services (as in "network services"), yes. I don't think it is going to be a service call, most likely it is going to be a low-priority message, because services (as in "RPC") generally map poorly to stateless distributed computing, as explained in the IDG.